sklearn/gaussian_process/tests/test_gaussian_process.py

```diff
@@ -66,8 +66,9 @@ def test_2d(regr=regression.constant, corr=correlation.squared_exponential,
 
     assert_true(np.allclose(y_pred, y) and np.allclose(MSE, 0.))
 
-    assert_true(np.all(gp.theta_ >= thetaL))  # Lower bounds of hyperparameters
-    assert_true(np.all(gp.theta_ <= thetaU))  # Upper bounds of hyperparameters
+    eps = np.finfo('float').eps
```

Comment:
It's better to use an explicit dtype (either 32 or 64 bit or use the dtype of the variable:

```python
eps = np.finfo(gp.theta_.dtype).eps
```